app/src/main/java/net/gsantner/markor/format/markdown/MarkdownTextActions.java

```diff
@@ -100,8 +100,8 @@ public void onClick(View view) {
                     runMarkdownRegularPrefixAction("### ");
                     break;
                 }
-                /*case R.string.tmaid_common_unordered_list_hyphen: {
```

Comment:
I updated the code here, even though it's commented, to keep things consistent if this is ever uncommented.